content/sensu-enterprise/3.3/integrations/email.md

```diff
@@ -37,11 +40,16 @@ ERB templating syntax][5]._
 
 ### Example(s) {#custom-email-templates-example}
```

Comment:
```suggestion
### Example(s) {#examples-custom-email-templates-example}
```

Reply:
There's a space being removed here, not sure why. I only wanted the `examples-` to be added on.